spec/features/checkout_feature_spec.rb

```diff
@@ -14,7 +14,7 @@
   after(:all) { DatabaseCleaner.clean_with(:truncation) }
 
   context "when not logged in" do
-    it "redirects to login page if not logged in", js: true do
+    it "redirects to login page if not logged in", :redirect_login, js: true do
```

Comment:
**Nitpick - Test description**
In the example description, we do not need `if not logged in` in the description. Reason being that there is a `context` with description `when not logged in`.

Hence, we could just have `it "redirects to the login page"`.